### PR TITLE
Fix version-converter to generate valid identifiers

### PR DESCRIPTION
**Pull request by gramalingam**
### Description
The version-converter generates invalid ONNX identifiers (like "1"). Change it to generate a valid identifier.

### CODE REVIEW

File: onnx/common/ir.h [hunk 1]
```diff
@@ -39,6 +39,16 @@
 
 namespace ONNX_NAMESPACE {
 
+namespace { // internal/private API
+
+std::string toVarName(size_t i) {
+  std::ostringstream oss;
+  oss << "_v_" << i;
```
Comment (justinchuby) [inline onnx/common/ir.h:46]: nit: Would it make sense to just use a formatting string for this?

**Review by justinchuby — APPROVED** [review]

File: onnx/common/ir.h (hunk 1 above)
Comment (gramalingam) [reply onnx/common/ir.h:46]: Which formatting string / API are you referring to?

Comment (justinchuby) [reply onnx/common/ir.h:46]: Looks like std::format isn’t until c++ 20. NVM 😅

Comment (gramalingam) [reply onnx/common/ir.h:46]: right

**Pull request merged**